test/read_config_test.go

```diff
@@ -47,7 +47,7 @@ func TestRead_EmptyTimeoutConfig(t *testing.T) {
 	wantPort := 8080
 
 	if config.Port != wantPort {
-		t.Logf("Port want: %d, got %d", wantPort, config.Port)
+		t.Logf("WatchdogPort want: %d, got %d", wantPort, config.Port)
```

Comment:
Was this intentional?

Reply:
Ah.. that's Goland IDE trying to be smart. I'll fix it.